configs/yolox/README.md

```diff
@@ -29,6 +29,26 @@ YOLOX-l model structure
 1. The test score threshold is 0.001.
 2. Due to the need for pre-training weights, we cannot reproduce the performance of the `yolox-nano` model. Please refer to https://github.com/Megvii-BaseDetection/YOLOX/issues/674 for more information.
 
+## 🥳 🚀 Update
+
+YOLOX uses a default training configuration of `8xbs8` which results in a long training time, we expect it to use `8xbs32` to speed up the training and not cause a decrease in mAP. I modified `train_batch_size_per_gpu` from 8 to 32, `batch_augments_interval` from 10 to 1 and `base_lr` from 0.01 to 0.04 under YOLOX-s default configuration based on the linear scaling rule, which resulted in mAP degradation. Finally, I found that using RTMDet's training hyperparameter can improve performance, which also validates the superiority of RTMDet's training hyperparameter.
```

Comment:
```suggestion
YOLOX uses a default training configuration of `8xbs8` which results in a long training time, we expect it to use `8xbs32` to speed up the training and not cause a decrease in mAP. We modified `train_batch_size_per_gpu` from 8 to 32, `batch_augments_interval` from 10 to 1 and `base_lr` from 0.01 to 0.04 under YOLOX-s default configuration based on the linear scaling rule, which resulted in mAP degradation. Finally, we found that using RTMDet's training hyperparameter can improve performance, which also validates the superiority of RTMDet's training hyperparameter.
```

Reply:
1. Need to illustrate the information about the training time change.
2. It would be better to put the section `Update` above the original one.